### ISSUE TITLE
Java regression failure - need to disable sigsegv handling in UCX/Java

### ISSUE THREAD

**Issue by Artemy-Mellanox** [openucx/ucx#3337]
```
03:09:17 -------------------------------------------------------
03:09:17  T E S T S
03:09:17 -------------------------------------------------------
03:09:17 Running org.ucx.jucx.UcpWorkerTest
03:09:18 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.672 sec
03:09:18 Running org.ucx.jucx.UcpContextTest
03:09:18 Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec
03:09:18 [hpc-test-node3:53370:0:53406] Caught signal 11 (Segmentation fault: invalid permissions for mapped object at address 0x7ffff7ff7680)
03:09:18 ==== backtrace ====
03:09:18  0 0x0000000000690604 JavaCalls::call_helper()  ???:0
03:09:18  1 0x00000000006912e1 JavaCalls::call_virtual()  ???:0
03:09:18  2 0x0000000000691787 JavaCalls::call_virtual()  ???:0
03:09:18  3 0x0000000000a747fd JavaThread::exit()  ???:0
03:09:18  4 0x0000000000a75472 JavaThread::thread_main_inner()  ???:0
03:09:18  5 0x0000000000a7568c JavaThread::run()  ???:0
03:09:18  6 0x0000000000926268 java_start()  ???:0
03:09:18  7 0x0000000000007e25 start_thread()  pthread_create.c:0
03:09:18  8 0x00000000000f834d __clone()  ???:0
03:09:18 ===================
03:09:18 Sending notification to artemyko@mellanox.com
03:09:19 [hpc-test-node3:53370:0:53406] Process frozen...
06:52:42 Build timed out (after 240 minutes). Marking the build as failed.
```

http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9182/label=vulcan-jenkins,worker=0/console
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9182/label=hpc-test-node3,worker=0/console

**Comment by petro-rudenko** [comment]
https://github.com/facebook/rocksdb/wiki/JNI-Debugging
```
You will likely need to instruct gdb not to stop on internal SIGSEGV and SIGBUS signals generated by the JVM:

(lldb) pro hand -p true -s false SIGSEGV
(lldb) pro hand -p true -s false SIGBUS
```
So first will try to disable signal handling on ucx: `ERROR_SIGNALS=no`

**Comment by yosefe** [comment]
@petro-rudenko is this expected JVM will generate such signals internally?
if yes - need to disable UCX signal handling from Java wrapper code

**Comment by petro-rudenko** [comment]
Sure: JVM will handle by itself: https://stackoverflow.com/questions/43345363/can-a-sigsegv-in-java-not-crash-the-jvm
```
Sure, if a SIGSEGV occurs while executing Java code (not native code) it is most likely due to dereferencing a Java null. That can be trapped and turned into a NullPointerException and "thrown". An application can recover from this; i.e. by "catching" the exception.
```

**Comment by petro-rudenko** [comment]
Errors happened after the tests finished (contexts cleanup called). But seems ucx still handle signals.

**Comment by yosefe** [comment]
@petro-rudenko can we close this?

**Comment by yosefe** [comment]
Fixed in #3392
